app/src/features/surveys/components/SurveyStratumForm.tsx

```diff
@@ -0,0 +1,313 @@
+import { mdiDotsVertical, mdiPencilOutline, mdiPlus, mdiTrashCanOutline } from '@mdi/js';
+import Icon from '@mdi/react';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import IconButton from '@mui/material/IconButton';
+import Card from '@mui/material/Card';
+import DialogActions from '@mui/material/DialogActions';
+import {  Formik, FormikProps, useFormikContext } from 'formik';
+import { IEditSurveyRequest } from 'interfaces/useSurveyApi.interface';
+import yup from 'utils/YupSchema';
+import { useState, useRef } from 'react';
+import { Box, Dialog, DialogContent, DialogContentText, DialogTitle, ListItemIcon, Menu, MenuItem, MenuProps, useMediaQuery, useTheme } from '@mui/material';
+import CustomTextField from 'components/fields/CustomTextField';
+
+interface IStratum {
+  /**
+   * @TODO We probably won't track the survey stratum ID in the frontend. Reason: new records won't have an ID.
+   * From a technical perspective, it is much easier to simply erase all stratum and re-insert them upon
```

Comment:
I've flip flopped a lot on this issue. We've got examples of both approaches in our codebase which complicates things a bit. No work needed just interesting to think about this

Reply:
I ended up removing this comment